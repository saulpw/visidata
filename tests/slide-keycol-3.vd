sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	OrderDate		key-col		!	toggle current column as a key column
sample	Region		key-col		!	toggle current column as a key column
sample	Rep		key-col		!	toggle current column as a key column
sample	Rep		slide-left		H	slide current column left
