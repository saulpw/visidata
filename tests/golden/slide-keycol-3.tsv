OrderDate	Rep	Region	Item	Units	Unit_Cost	Total
2016-01-06	Jones	East	Pencil	95	1.99	189.05
2016-01-23	Kivell	Central	Binder	50	19.99	999.50
2016-02-09	Jardine	Central	Pencil	36	4.99	179.64
2016-02-26	Gill	Central	Pen	27	19.99	539.73
2016-03-15	Sorvino	West	Pencil	56	2.99	167.44
2016-04-01	Jones	East	Binder	60	4.99	299.40
2016-04-18	Andrews	Central	Pencil	75	1.99	149.25
2016-05-05	Jardine	Central	Pencil	90	4.99	449.10
2016-05-22	Thompson	West	Pencil	32	1.99	63.68
2016-06-08	Jones	East	Binder	60	8.99	539.40
2016-06-25	Morgan	Central	Pencil	90	4.99	449.10
2016-07-12	Howard	East	Binder	29	1.99	57.71
2016-07-29	Parent	East	Binder	81	19.99	1619.19
2016-08-15	Jones	East	Pencil	35	4.99	174.65
2016-09-01	Smith	Central	Desk	2	125.00	250.00
2016-09-18	Jones	East	Pen Set	16	15.99	255.84
2016-10-05	Morgan	Central	Binder	28	8.99	251.72
2016-10-22	Jones	East	Pen	64	8.99	575.36
2016-11-08	Parent	East	Pen	15	19.99	299.85
2016-11-25	Kivell	Central	Pen Set	96	4.99	479.04
2016-12-12	Smith	Central	Pencil	67	1.29	86.43
2016-12-29	Parent	East	Pen Set	74	15.99	1183.26
2017-01-15	Gill	Central	Binder	46	8.99	413.54
2017-02-01	Smith	Central	Binder	87	15.00	1305.00
2017-02-18	Jones	East	Binder	4	4.99	19.96
2017-03-07	Sorvino	West	Binder	7	19.99	139.93
2017-03-24	Jardine	Central	Pen Set	50	4.99	249.50
2017-04-10	Andrews	Central	Pencil	66	1.99	131.34
2017-04-27	Howard	East	Pen	96	4.99	479.04
2017-05-14	Gill	Central	Pencil	53	1.29	68.37
2017-05-31	Gill	Central	Binder	80	8.99	719.20
2017-06-17	Kivell	Central	Desk	5	125.00	625.00
2017-07-04	Jones	East	Pen Set	62	4.99	309.38
2017-07-21	Morgan	Central	Pen Set	55	12.49	686.95
2017-08-07	Kivell	Central	Pen Set	42	23.95	1005.90
2017-08-24	Sorvino	West	Desk	3	275.00	825.00
2017-09-10	Gill	Central	Pencil	7	1.29	9.03
2017-09-27	Sorvino	West	Pen	76	1.99	151.24
2017-10-14	Thompson	West	Binder	57	19.99	1139.43
2017-10-31	Andrews	Central	Pencil	14	1.29	18.06
2017-11-17	Jardine	Central	Binder	11	4.99	54.89
2017-12-04	Jardine	Central	Binder	94	19.99	1879.06
2017-12-21	Andrews	Central	Binder	28	4.99	139.72
